python/ray/utils.py

```diff
@@ -97,6 +98,7 @@ def push_error_to_driver_through_redis(redis_client,
     """
     if driver_id is None:
         driver_id = ray.ObjectID.nil_id()
+    # TODO(suquark): How should we use data here?
```

Comment:
Hm, it isn't part of the `ErrorTableData` schema, so maybe we should just remove it. Do we pass in the `data` argument anywhere?

Reply:
We pass `data` in some tests.

Reply:
So will we not use `data` any more? Or is it just not implemented in the `ErrorTableData` schema?

Reply:
I think it's ok to remove given that it's not actually used. It looks like we pass it into `push_error_to_driver_through_redis` or `push_error_to_driver`, but then it never comes out, right?